fastify.js

```diff
@@ -96,6 +96,7 @@ function fastify (options) {
   const requestIdLogLabel = options.requestIdLogLabel || 'reqId'
   const bodyLimit = options.bodyLimit || defaultInitOptions.bodyLimit
   const disableRequestLogging = options.disableRequestLogging || false
+  const exposeHeadRoutes = options.exposeHeadRoutes != null ? options.exposeHeadRoutes : true
```

Comment:
This needs docs

Reply:
If the default is false, we can land this as minor against master.

Reply:
I'm fine with both approaches, setting it as `false` by default will require way fewer changes to the tests as most of them are broken because of the feature. Should we move forward with a minor version?

Reply:
+1 for default to false.

Reply:
Done in 124fa9e
I'll just need to adjust my PR to point to `master`